packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/docs/continuous_delivery.md

```diff
@@ -0,0 +1,106 @@
+## Fastlane
+
+### Distribution repository
+
+Create a new Github repository used for storing files and credentials for distribution of your app.
+Clone the repository and open the directory in the terminal.
+Using the [rx_bloc_cli][rx_bloc_cli_link] package, execute the following command:
+
+```
+rx_bloc_cli create_distribution .
+```
+
+The `create_distribution` command will populate the directory with the file structure required for things to work.
+The generated `README` file contains steps on where to download and how to name the required files.
+Within the `android` and `ios` directories, paste the previously downloaded files and rename them accordingly.
+
+You should also set the `MOBILE_DISTRIBUTION_ENCRYPTION_PASSWORD` variable in your terminal.
+This password will be used for encrypting/decrypting your files within the newly created repository.
+
+Before committing any changes, make sure you encrypt all the files using the `encode.sh` script.
+
+### Fastfile amendments
+
+Inside the `{project_root}/fastlane/Fastfile`, you need to make changes to some variables in order for the configuration to work.
+
+Replace the value of the `APPLE_DEVELOPMENT_TEAM` variable with your own [Apple Development Team ID][apple_development_team_id].
+
+When downloading the `.p8` auth key, the file name should have the following format: `AuthKey_{KEY_ID}.p8` where the `KEY_ID` is your unique id.
+Take that value and paste it into the `IOS_P8_AUTH_KEY_ID` variable.
+
+The value of the `IOS_P8_AUTH_KEY_ISSUER_ID` variable is the value retrieved from the `contentProviderPublicId` key within [this page][apple_issuer_id_details].
+If more than one `contentProviderPublicId` with different values is present, make sure to use the one from the appropriate organization account name.
+
+For each of the supported flavors, a separate firebase project will be required.
```

Comment:
"firebase project" this is incorrect. In one of our mobile application we have
1. One project for the production environment
2. Second project for all testing environments (dev, sit, uat)